dot/core/service.go

```diff
@@ -59,10 +57,6 @@ type Service struct {
 	net              Network
 	digestHandler    DigestHandler
 
-	// Current runtime and hash of the current runtime code
-	rt       runtime.Instance
```

Comment:
Should `Runtime` also be removed from `Config` struct since the service is no longer storing it?

Reply:
The test is using this runtime. I have created another issue to remove the runtime. https://github.com/ChainSafe/gossamer/issues/1674